index.html

```diff
@@ -636,10 +636,10 @@ <h3>BitstringStatusListCredential</h3>
 minimum two properties:
                 <ul>
                   <li id="status">
-`status`, being a string of the hex value of the status
+`status`, being a machine-readable string of the hex value of the status
                   </li>
                   <li id="message">
-`message`, being a string containing the associated message
+`message`, being a human-readable string containing the associated message
```

Comment:
We'd like to steer clear of "messages" and the sense that this is a display string. Perhaps:

```suggestion
`message`, being an informative string to assist with debugging
```